src/relayserver/RelayServer.ts

```diff
@@ -299,12 +307,47 @@ export class RelayServer extends EventEmitter {
     process.exit(1)
   }
 
+  trustedPaymastersGasLimits: {[key: string]: PaymasterGasLimits} = {}
```

Comment:
Please do not define class member fields on the line 310. I know we are not in Pascal times but having methods defined in the beginning of the file is very helpful IMO.

Also, I prefer to have `Map` to typed `object`. Indexing operator is cool but somehow it does not feel right...